open_academy/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "open_academy",
+    'license': "LGPL-3",
+
+    'summary': """
+        Short (1 phrase/line) summary of the module's purpose, used as
+        subtitle on modules listing or apps.openerp.com""",
+
+    'author': "Vauxoo",
+    'website': "http://www.yourcompany.com",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Uncategorized',
+    'version': '15.0.1.0.0',
+
+    # any module necessary for this one to work correctly
+    'depends': ['base'],
+
+    # always loaded
+    'data': [
+        # Check the order, 1. security, 2 views, 3 menus
+        'security/ir.model.access.csv',
+        'views/course_views.xml',
```

Comment:
Please rename this file. See [guideline](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming) for file naming please. Your model is` openacademy.course` so this file should be

```suggestion
        'views/open_academy_course_views.xml',
```